File: src/main/java/toast/client/commands/cmds/GuiReset.java
package toast.client.commands.cmds;

import toast.client.ToastClient;
import toast.client.commands.Command;
import toast.client.utils.Logger;

import static toast.client.ToastClient.clickGui;
import static toast.client.ToastClient.clickGuiHasOpened;

public class GuiReset extends Command {
    public GuiReset() {
        super("GuiReset", ToastClient.cmdPrefix + "guireset", "Regenerates clickgui category positions according to current screen size", false, "guireset", "gr");
    }

    public void run(String[] args) {
        if(clickGuiHasOpened) {
            clickGui.getSettings().initCategoryPositions();
            clickGui.getSettings().savePositions();
            Logger.message("Re-arranged ClickGui, if you still have problems, try setting your gui scale to minimum.", Logger.INFO, true);
        }else{
            Logger.message("ClickGui hasn't been opened!", Logger.ERR, true);
        }
    }
}

File: src/main/java/toast/client/commands/cmds/GuiReset.kt
package toast.client.commands.cmds

import toast.client.ToastClient
import toast.client.commands.Command
import toast.client.utils.MessageUtil

/**
 * Command to regenerate ClickGui category positions according to current window resolution
 */
class GuiReset : Command("GuiReset", "${ToastClient.cmdPrefix}guireset", "Makes ClickGui fit on screen as much as possible", false, "guireset", "gr", "fixgui") {
    override fun run(args: Array<String>) {
        if (ToastClient.clickGuiHasOpened) {
            ToastClient.clickGui.settings.initCategoryPositions()
            ToastClient.clickGui.settings.savePositions()
            MessageUtil.sendMessage("Re-arranged ClickGui, if you still have problems, try setting your gui scale to minimum.", MessageUtil.Color.GREEN)
        } else {
            MessageUtil.sendMessage("ClickGui hasn't been opened!", MessageUtil.Color.RED)
        }
    }
}